packages/suite/src/views/wallet/coinmarket/common/CoinmarketFeaturedOffers/CoinmarketFeaturedOffers.tsx

```diff
@@ -0,0 +1,60 @@
+import CoinmarketFeaturedOffersItem, {
+    CoinmarketFeaturedOffersSkeletonItem,
+} from './CoinmarketFeaturedOffersItem';
+import { isCoinmarketExchangeOffers } from 'src/hooks/wallet/coinmarket/offers/useCoinmarketCommonOffers';
+import { getBestRatedQuote } from 'src/utils/wallet/coinmarket/coinmarketUtils';
+import { WalletSubpageHeading } from 'src/components/wallet';
+import { ExchangeTrade } from 'invity-api';
+import { useCoinmarketFormContext } from 'src/hooks/wallet/coinmarket/form/useCoinmarketCommonForm';
+import CoinmarketFeaturedOffersEmpty from './CoinmarketFeaturedOffersEmpty';
+
+const CoinmarketFeaturedOffers = () => {
+    const context = useCoinmarketFormContext();
+    const {
+        type,
+        form: { state },
+    } = context;
+    const quotes = context?.quotes ?? [];
+    const featuredQuotes = quotes.filter(quote => quote.infoNote);
+    const noFeaturedOffers = featuredQuotes.length === 0;
+    const bestRatedQuote = getBestRatedQuote(quotes, type);
+
+    // for now do not show featured offers on exchange
+    if (isCoinmarketExchangeOffers(context)) return null;
+
+    if (!state.isFormLoading && !state.isFormInvalid && noFeaturedOffers) return null;
+
+    let content;
+    if (state.isFormInvalid) {
+        content = <CoinmarketFeaturedOffersEmpty />;
+    } else if (state.isFormLoading) {
+        content = (
+            <>
+                <CoinmarketFeaturedOffersSkeletonItem />
+                <CoinmarketFeaturedOffersSkeletonItem />
+            </>
+        );
+    } else {
+        content = featuredQuotes.map(_quote => {
+            const quote = _quote as Exclude<typeof _quote, ExchangeTrade>; // for now only on sell and buy
+
+            return (
+                <CoinmarketFeaturedOffersItem
+                    key={quote?.orderId}
+                    context={context}
+                    quote={quote}
+                    isBestRate={bestRatedQuote?.orderId === quote?.orderId}
+                />
+            );
+        });
+    }
```

Comment:
After discussing with @marek-andrsk, we decided to remove the loading and error states. Please replace them with successful featured offers, if any, and delete the related code.